mailbox/elasticsearch/src/test/resources/logback-test.xml

```diff
@@ -1,26 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <configuration>
 
-        <contextListener class="ch.qos.logback.classic.jul.LevelChangePropagator">
-                <resetJUL>true</resetJUL>
-        </contextListener>
-
-        <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
-                <encoder>
-                        <pattern>%d{HH:mm:ss.SSS} [%-5level] %logger{15} - %msg%n%rEx</pattern>
-                        <immediateFlush>false</immediateFlush>
-                </encoder>
-        </appender>
-
-        <root level="ERROR">
-                <appender-ref ref="CONSOLE" />
-        </root>
-
-
-        <logger name="org.apache.james" level="DEBUG" >
-                <appender-ref ref="CONSOLE" />
-        </logger>
-
-        <logger name="com.datastax.driver.core.QueryLogger.SLOW" level="DEBUG" />
```

Comment:
you don't keep this one?

Reply:
it's test logging, we don't care about cassandra slow logs, isn't it ?

Reply:
at first I though it could be a good idea to keep them, to see if we are doing some bad things with cassandra. But maybe the test environment is not suitable for that